server/apps/archive/archive_link.py

```diff
@@ -24,8 +25,8 @@ class ArchiveLinkResource(Resource):
     resource_title = endpoint_name
 
     schema = {
-        'link_id': Resource.rel('archive', False, required=False),
-        'desk': Resource.rel('desks', required=False)
+        'link_id': Resource.rel('archive', False, type='string'),
+        'desk': Resource.rel('desks', False)
```

Comment:
It might be a good idea to retain the `required` argument keyword so that's it's immediately clear what that `False` argument stands for.